profiles.md

```diff
@@ -0,0 +1,208 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Single Profile
+**GET /api/cris/profiles/<:eperson-uuid>**
+
+Provide information about a specific profile. Only the Administrators or the owner eperson can access this endpoint.
+
+The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "visible": true,
+  "type": "profile"
+}
+```
+
+Attributes:
+* id: the profile id that corresponds to the id of the eperson associated
+* visible: indicates whether the profile is public (visible: true) or private (otherwise)
+* type: the resource type (profile)
+
+Exposed links:
+* item: the item with which the profile is modeled
+* eperson: the eperson related to the profile
+
+It would respond with:
+* 200 OK - Returning the profile if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the profile
+* 404 Not found - if the profile doesn't exist
+
+## Linked entities
+
+### Item
+**GET /api/cris/profiles/<:eperson-uuid>/item**
+
+Returns the item that model the profile of the eperson with the given uuid. The JSON response document is as follow
+
+```json
+{
+  "id": "d0601db5-5ceb-4f8d-8762-bd5b3b97845d",
+  "uuid": "d0601db5-5ceb-4f8d-8762-bd5b3b97845d",
+  "name": null,
+  "handle": "123456789/511",
+  "metadata": {
+    "cris.owner": [{
+        "value": "Mortimer Smith",
+        "language": null,
+        "authority": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+        "confidence": 600,
+        "place": 0
+      }],
+    "cris.sourceId": [{
```

Comment:
I'm not sure of the purpose of this field, but again we don't have a "cris" schema. I'm also hesitant to create a "cris" schema, as I don't think the idea of a Researcher Profile is specific to a CRIS use case.  So, I think we may need a different name, if this field is needed.

Reply:
We can remove this metadata.